### PR TITLE
add tests for shared browser hold_browser_open config

### PR DESCRIPTION
**Pull request by aleksandr-kotlyar**
Increased code coverage of property [`hold_browser_open`](
https://codecov.io/gh/yashaka/selene/compare/0e7ed43957d08ff37f37c731e8fc9136500b85c4...c9e38e874581b6aca022662ed1763ae74f579041/src/selene/support/shared/config.py)
image

### CODE REVIEW

**Comment by aleksandr-kotlyar** [comment]
May be we should move them to integration tests.

File: tests/acceptance/shared_browser/hold_open_test.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from selene import have
+from selene.support.shared import config, browser, SharedConfig
+
+
+def setup_module():
+    config.hold_browser_open = True
```
Comment (yashaka) [inline tests/acceptance/shared_browser/hold_open_test.py:6]: hold_browser_open_on_session_exit - this is what is meant under hold_browser_open....

Comment (yashaka) [reply tests/acceptance/shared_browser/hold_open_test.py:6]: that's why test cases below do not actually test this config option....

**Review by yashaka — CHANGES_REQUESTED** [review]
current implementation does not reach the goal...

actually I have no idea how to reach it... 
maybe you can think more and find a way :)

maybe we need here some kind of unit test...

File: tests/acceptance/shared_browser/hold_open_test.py (hunk 1 above)
Comment (aleksandr-kotlyar) [reply tests/acceptance/shared_browser/hold_open_test.py:6]: But look at the tests.
They test if browser opened through one test session. That's covers the scenario.

But you are right. Scenario with browser open after pytest session will be nice to have ;)
I think how to achieve that test, and my thoughts are: somehow get id of webdriver and browser processes in pytest session, write them down to the file in the end of session. And after session execute python script which will check system processes trying to find these ids... and if they will be found - test will pass, and thats will be the test you meant.

**Comment by aleksandr-kotlyar** [comment]
> current implementation does not reach the goal...
> 
> actually I have no idea how to reach it...
> maybe you can think more and find a way :)
> 
> maybe we need here some kind of unit test...

I have made a research through selenide repo to [find all info about their usage of holdBrowserOpen](https://github.com/search?q=org%3Aselenide+holdBrowserOpen&type=commits) and found how @selenide has reached it https://github.com/selenide/selenide/commit/e5489722711c49fb3f435ca12ffdbe1c503b8ce8
#### Todo
Answer a question can we make such tests selenide/selenide@e548972 for selene on python, are they relevant for selene realization?

**Pull request merged**